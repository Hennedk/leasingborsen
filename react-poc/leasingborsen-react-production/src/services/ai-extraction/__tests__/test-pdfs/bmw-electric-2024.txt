BMW Erhvervsleasing 2024
=======================

Gyldig periode: 15. maj - 31. december 2024
Alle priser er ekskl. 25% moms

BMW iX3 - Fuldelektrisk SUV
===========================

Batterisystem: 74 kWh
Rækkevidde (WLTP): 458 km
Opladning: DC hurtigladning op til 150 kW
Motoreffekt: 286 hk / 210 kW
Drejningsmoment: 400 Nm

iX3 xDrive30 - 286 hk
Månedsydelse: 8.500 kr (ekskl. moms)
Udbetaling: 85.000 kr
Løbetid: 36 måneder
Årlig kørsel: 20.000 km
CO2-udslip: 0 g/km
Energimærke: A++
0-100 km/t: 6,8 sekunder

BMW X3 - Plugin Hybrid
======================

X3 xDrive30e M Sport Steptronic - 292 hk
Kombination af: 2,0-liter TwinPower Turbo + El-motor
Total systemeffekt: 292 hk / 215 kW
Elektrisk rækkevidde: 46 km
Batteristørrelse: 12 kWh

Månedsydelse: 9.200 kr (ekskl. moms)
Udbetaling: 92.000 kr
Løbetid: 36 måneder
Årlig kørsel: 20.000 km
CO2-udslip: 36 g/km
Energimærke: A+
Brændstofforbrug: 2,2 l/100 km
0-100 km/t: 6,0 sekunder

BMW 3-Serie - Konventionelle motorer
====================================

320i Advantage Steptronic - 184 hk
Motor: 2,0-liter TwinPower Turbo benzin
Transmission: 8-trins Steptronic automatgear

Månedsydelse: 6.200 kr (ekskl. moms)
Udbetaling: 62.000 kr
Løbetid: 36 måneder
Årlig kørsel: 20.000 km
CO2-udslip: 134 g/km
Energimærke: C
Brændstofforbrug: 17,2 km/l
0-100 km/t: 7,1 sekunder

320d Advantage Steptronic - 190 hk  
Motor: 2,0-liter TwinPower Turbo diesel
Transmission: 8-trins Steptronic automatgear

Månedsydelse: 6.800 kr (ekskl. moms)
Udbetaling: 68.000 kr
Løbetid: 36 måneder
Årlig kørsel: 20.000 km
CO2-udslip: 127 g/km
Energimærke: B
Brændstofforbrug: 20,4 km/l
0-100 km/t: 6,8 sekunder

Tilvalg og pakker:
=================

Teknologipakker:
- BMW Live Cockpit Professional: 650 kr/måned
- Driving Assistant Professional: 450 kr/måned
- Parking Assistant Plus: 350 kr/måned

Komfortpakker:
- M Sportspakke: 750 kr/måned
- Comfort Access: 250 kr/måned
- Panoramatag: 450 kr/måned

Hjul og dæk:
- M Alufælge 19": 400 kr/måned
- Vinterhjulssæt 18": 350 kr/måned
- Runflat dæk: 200 kr/måned

Vilkår:
=======

- Priser ekskl. 25% moms
- Erhvervsleasing kun for virksomheder
- Minimum løbetid: 24 måneder
- Maximum løbetid: 60 måneder
- Leveringstid: 3-6 måneder
- Servicepakke kan tilkøbes

Kontakt:
========

BMW Danmark A/S
Erhvervsleasing
Telefon: 70 25 75 20
E-mail: erhverv@bmw.dk

Dokumentinfo:
=============
Udstedt: 15. maj 2024
Type: Erhvervsleasing prisliste  
Valuta: DKK
Målgruppe: Erhvervskunder