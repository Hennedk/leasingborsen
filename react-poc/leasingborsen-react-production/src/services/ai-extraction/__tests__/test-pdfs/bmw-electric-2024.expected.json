{
  "documentInfo": {
    "brand": "BMW",
    "documentDate": "2024-05-15",
    "currency": "DKK",
    "language": "da",
    "documentType": "business_leasing"
  },
  "vehicles": [
    {
      "model": "iX3",
      "category": "SUV",
      "leasePeriodMonths": 36,
      "powertrainType": "electric",
      "variants": [
        {
          "variantName": "xDrive30",
          "engineSpecification": "Electric - 286 hk",
          "transmission": "automatic",
          "pricing": {
            "monthlyPayment": 8500,
            "firstPayment": 85000,
            "annualKilometers": 20000
          },
          "specifications": {
            "co2EmissionsGkm": 0,
            "energyLabel": "A++",
            "electricRangeKm": 458,
            "batteryCapacityKwh": 74,
            "horsePower": 286,
            "acceleration0to100": 6.8
          }
        }
      ]
    },
    {
      "model": "X3",
      "category": "SUV",
      "leasePeriodMonths": 36,
      "powertrainType": "plugin_hybrid",
      "variants": [
        {
          "variantName": "xDrive30e M Sport Steptronic",
          "engineSpecification": "2.0 TwinPower Turbo + Electric - 292 hk",
          "transmission": "automatic",
          "pricing": {
            "monthlyPayment": 9200,
            "firstPayment": 92000,
            "annualKilometers": 20000
          },
          "specifications": {
            "fuelConsumptionKmpl": 45.5,
            "co2EmissionsGkm": 36,
            "energyLabel": "A+",
            "electricRangeKm": 46,
            "batteryCapacityKwh": 12,
            "horsePower": 292,
            "acceleration0to100": 6.0
          }
        }
      ]
    },
    {
      "model": "3-Serie",
      "category": "Sedan",
      "leasePeriodMonths": 36,
      "powertrainType": "gasoline",
      "variants": [
        {
          "variantName": "320i Advantage Steptronic",
          "engineSpecification": "2.0 TwinPower Turbo - 184 hk",
          "transmission": "automatic",
          "pricing": {
            "monthlyPayment": 6200,
            "firstPayment": 62000,
            "annualKilometers": 20000
          },
          "specifications": {
            "fuelConsumptionKmpl": 17.2,
            "co2EmissionsGkm": 134,
            "energyLabel": "C",
            "horsePower": 184,
            "acceleration0to100": 7.1
          }
        },
        {
          "variantName": "320d Advantage Steptronic",
          "engineSpecification": "2.0 TwinPower Turbo Diesel - 190 hk",
          "transmission": "automatic",
          "pricing": {
            "monthlyPayment": 6800,
            "firstPayment": 68000,
            "annualKilometers": 20000
          },
          "specifications": {
            "fuelConsumptionKmpl": 20.4,
            "co2EmissionsGkm": 127,
            "energyLabel": "B",
            "horsePower": 190,
            "acceleration0to100": 6.8
          }
        }
      ]
    }
  ],
  "accessories": [
    {
      "packageName": "BMW Live Cockpit Professional",
      "monthlyCost": 650,
      "category": "other"
    },
    {
      "packageName": "Driving Assistant Professional",
      "monthlyCost": 450,
      "category": "other"
    },
    {
      "packageName": "Parking Assistant Plus",
      "monthlyCost": 350,
      "category": "other"
    },
    {
      "packageName": "M Sportspakke",
      "monthlyCost": 750,
      "category": "other"
    },
    {
      "packageName": "Comfort Access",
      "monthlyCost": 250,
      "category": "other"
    },
    {
      "packageName": "Panoramatag",
      "monthlyCost": 450,
      "category": "other"
    },
    {
      "packageName": "M Alufælge 19\"",
      "monthlyCost": 400,
      "category": "wheels"
    },
    {
      "packageName": "Vinterhjulssæt 18\"",
      "monthlyCost": 350,
      "category": "wheels"
    },
    {
      "packageName": "Runflat dæk",
      "monthlyCost": 200,
      "category": "wheels"
    }
  ],
  "metadata": {
    "extractionTimestamp": "2024-06-24T12:00:00.000Z",
    "documentPages": 1,
    "extractionWarnings": []
  }
}